src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+    protected String filePath;
+    protected static File taskList;
+
+    public Storage(String filePath) {
```

Comment:
Perhaps need a descriptive header comments for the Storage method? 🤔
And I noticed the same issue in several other places below.